Move importing `core` to the bottom to fix AttributeError

DableUTeeF opened the pull request:
Apparently in my machine I always got `AttributeError: module 'mmdet2trt' has no attribute 'core'` when importing the module, this happens with both
```
import mmdet2trt
```
and
```
from mmdet2trt import mmdet2trt
```

After some investigation I found out that if I run
```
import mmdet2trt.core
```
I got `AttributeError: module 'mmdet2trt' has no attribute 'ops'`, but
```
import mmdet2trt.ops
```
Worked fine, and after move `from .core import *` to the bottom the thing works

grimoire commented:
Hi, Thanks for your contribution.
Could you fix the [checks failing](https://github.com/grimoire/mmdetection-to-tensorrt/pull/64/checks) of this PR?

DableUTeeF commented:
> Hi, Thanks for your contribution.
> Could you fix the [checks failing](https://github.com/grimoire/mmdetection-to-tensorrt/pull/64/checks) of this PR?

The `isort` itself is the problem. And that's exactly what I fixed in this PR.

As for `trailing-whitespace`. That's weird since the only file I changed is the `__init__` and I can't find any of that.

grimoire commented:
You can ignore the `trailing-whitespace`, it is caused by another PR.
Current `master` branch works fine on my device and most other user's device. We better work together to find out why this error happened on your device before we merge this.
I believe there must be a way to fix your problem without break the CI.

mmeendez8 commented:
I was getting the same error. Fix worked perfectly for me, thanks :+1:

grimoire commented:
@DableUTeeF  @mmeendez8 
Could you please tell me the python/conda version you are using? I want to reproduce the error on my device.
And can you try the `__init__` below and see if the error still exist?
```python
from .converters import *  # noqa: F401,F403
from .mmdet2trt import Int8CalibDataset, mask_processor2trt, mmdet2trt

__all__ = ['Int8CalibDataset', 'mask_processor2trt', 'mmdet2trt']
```

Import everything in the root init might be a mistake...

mmeendez8 commented:
Yes, that works for me too!

Im using Python 3.6:
```
Python 3.6.9 (default, Jan 26 2021, 15:33:00) 
[GCC 8.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
```

DableUTeeF commented:
I was using python 3.6 from minicoda and the latest master branch of MMDetectionI'll try this when I'm at my PCSent from my Huawei phone-------- Original message --------From: "q.yao" ***@***.***>Date: Mon, May 24, 2021, 21:00To: grimoire/mmdetection-to-tensorrt ***@***.***>Cc: Natthapawn Hongcharoen ***@***.***>, Mention ***@***.***>Subject: Re: [grimoire/mmdetection-to-tensorrt] Move importing `core` to the bottom to fix AttributeError (#64)
@DableUTeeF  @mmeendez8
Could you please tell me the python/conda version you are using? I want to reproduce the error on my device.
And can you try the __init__ below and see if the error still exist?
from .converters import *  # noqa: F401,F403
from .mmdet2trt import Int8CalibDataset, mask_processor2trt, mmdet2trt

__all__ = ['Int8CalibDataset', 'mask_processor2trt', 'mmdet2trt']
Import everything in the root init might be a mistake...

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

grimoire commented:
I am using python 3.7.
@DableUTeeF If remove unnecessary import statement could work on your device, would you mind doing this fix for me?

DableUTeeF commented:
I committed a new one.

grimoire reviewed (approved):
LGTM!
